### ISSUE TITLE
Docker: basic example not working 

### ISSUE THREAD

**Issue by 1dal** [postmanlabs/newman#1532]
1. Newman Version (can be found via `newman -v`): postman/**newman_alpine33**:3.9.3
 2. OS details (type, version, and architecture): Debian 9 (Stretch), x64
 3. Are you using Newman as a library, or via the CLI? CLI
 3. Did you encounter this recently, or has this bug always been there: i started with Newman in Docker today
 4. Expected behaviour: newman is executed...
 5. Command / script used to run Newman:
```bash
docker run -t postman/newman_alpine33:3.9.3 --url="https://www.getpostman.com/collections/8a0c9bc08f062d12dcda"
```
 6. Sample collection, and auxiliary files (minus the sensitive details): `https://www.getpostman.com/collections/8a0c9bc08f062d12dcda`
 7. Screenshots (if applicable): -

Steps to reproduce the problem:

1. open terminal
2. copy example command from documentation
3. hit enter

**Comment by kunagpal** [comment]
@1dal This will be fixed with 3.9.4, which is scheduled to go out later this week.

**Comment by 1dal** [comment]
@kunagpal: hi, it is working, thank you for quick response
